src/NumericalAlgorithms/Interpolation/Callbacks/FindApparentHorizon.hpp

```diff
@@ -0,0 +1,171 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <utility>
+
+#include "ApparentHorizons/FastFlow.hpp"
+#include "DataStructures/DataBox/DataBox.hpp"
+#include "Informer/Verbosity.hpp"
+#include "Parallel/Abort.hpp"
+#include "Parallel/ConstGlobalCache.hpp"
+#include "Parallel/Invoke.hpp"
+#include "Parallel/Printf.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/PrettyType.hpp"
+
+/// \cond
+namespace StrahlkorperTags {
+template <typename Frame>
+struct Strahlkorper;
+}  // namespace StrahlkorperTags
+namespace db {
+template <typename TagsList>
+class DataBox;
+}  // namespace db
+namespace intrp {
+template <class Metavariables, typename InterpolationTargetTag>
+struct InterpolationTarget;
+namespace Tags {
+template <typename Metavariables>
+struct TemporalIds;
+}  // namespace Tags
+}  // namespace intrp
+namespace Tags {
+struct FastFlow;
+struct Verbosity;
+}  // namespace Tags
+template <typename Frame>
+class Strahlkorper;
+/// \endcond
+
+namespace intrp {
+namespace callbacks {
+
```

Comment:
you need to either document `callbacks` or put the following documentation into a group